cfme/tests/cli/test_appliance_console.py

```diff
@@ -1329,9 +1329,12 @@ def test_appliance_console_apache_reload_log_rotate():
     pass
 
 
-@pytest.mark.manual
 @pytest.mark.tier(1)
-def test_appliance_console_datetime_negative():
+@pytest.mark.automates([1790311])
+@pytest.mark.parametrize("invalid_date, invalid_time", [("202020-12-3", "25:13:01"),
+                                                        ("2020-13-3", "22:61:01"),
+                                                        ("2020-12-35", "24:11:61")])
```

Comment:
@dgaikwad After thinking on this a bit more, due to the way that the test is written. I think the parameters here should be the invalid_date vs the invalid time. Here's what I have in mind:

```python,
@pytest.mark.parametrize("invalid", ["invalid_date", "invalid_time"])
def test_appliance_console_datetime_negative(appliance, invalid):
    """
    polarion docblock
    """
    if invalid == "invalid_date":
        invalid_dates = <list of invalid dates>
        cmd_sets = [ ("ap", RETURN, "3", "Y", date) for date in invalid_dates]
   else:
        invalid_times = <list of invalid times>
        cmd_sets = [("ap", RETURN, "3", "Y", str(fauxfactory.gen_date()), time) for time in invalid_times]
   for cmd in cmd_sets:
        # now run each command
        result = appliance.appliance_console.run_commands(cmd, timeout=30, output=True)
        assert <blahblahblah>
```
What do you think about this structure? I think it makes the parameters of the test a bit more clear while also reducing code duplication.

Reply:
I agree on generating the invalid_dates rather than hardcoding them. Good suggestion overall.